Description: Upstream changes introduced in version 2010.12~rc3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 u-boot (2010.12~rc3-1) unstable; urgency=low
 .
   * New upstream release candidate.
   * Add dockstar target.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <clint@gnu.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- u-boot-2010.12~rc3.orig/snapshot.commit
+++ u-boot-2010.12~rc3/snapshot.commit
@@ -1 +1 @@
-3ac45988ab1605baaf83bd3acb80a197c9a208c2  Fri, 17 Dec 2010 10:15:38 +0100
+$Format:%H  %cD$
